docs: document slate heading as anchor links in user-manual (#6750)
plone/volto: docs/source/user-manual/blocks.md

To rearrange blocks, to the right of the block you want to move, click on its drag handle, move the block where you want it in the page, and release the drag handle.

(copy-anchor-links-label)=

### Copy slate headings as anchor links

When you move your mouse over a heading, a link icon appears on the right side. Click this icon to copy a direct link to that specific heading on the page.

> These links are useful for sharing a specific section of a page with others.

Reviewer: ```suggestion
Now you can paste the copied link to share a specific section of a page with others.
```

```{note}

> This feature is enabled by default and can be used only when user is loggedin.

Reviewer: ```suggestion
This feature is enabled by default, and only authenticated users can use this feature.
```
